File: src/python/tutorias/domain/estudiante.py
from .usuario import Usuario


class Estudiante(Usuario):
    def __init__(self, id_usuario: int, nombre: str, correo: str, carrera: str):
        super().__init__(id_usuario, nombre, correo)
        self._carrera = carrera

    def solicitar_tutoria(self):
        pass

    def cancelar_reserva(self):
        pass

File: src/java/edu/uees/tutorias/domain/Estudiante.java
package edu.uees.tutorias.domain;

public class Estudiante extends Usuario {

    private String carrera;

    public Estudiante(
            int id,
            String nombre,
            String correo,
            String carrera) {

        super(id, nombre, correo);
        this.carrera = carrera;
    }

    public void solicitarTutoria() {
        // La lógica se coordinará posteriormente desde ServicioReservas.
    }

    public void cancelarReserva() {
        // La lógica se coordinará posteriormente desde ServicioReservas.
    }
}